### PR TITLE
html file added

### PR DESCRIPTION
**Pull request by apragalatha**

### CODE REVIEW

File: FirstHtmlPage.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    <h1>Hi</h1>
```
Comment (NithyaRathnam) [inline FirstHtmlPage.html:9]: Can you add a title?

Comment (apragalatha) [reply FirstHtmlPage.html:9]: ok. will do.

File: FirstHtmlPage.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
```
Comment (NithyaRathnam) [inline FirstHtmlPage.html:8]: asdfas

File: FirstHtmlPage.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    <h1>Hi</h1>
+</body>
+</html>
```
Comment (NithyaRathnam) [inline FirstHtmlPage.html:11]: asdfas

**Review by NithyaRathnam — APPROVED** [review]

**Pull request merged**